src/molecules/form-fields/select-form-field.test.tsx

```diff
@@ -1,9 +1,171 @@
-// import React from "react";
-// import { render } from "@testing-library/react";
-// import { SelectFormField } from "./select-form-field";
-// import faker from "faker";
-// import uuid from "uuid";
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import {
+    SelectFormField,
+    InvalidSelectFormValueClass,
+} from "./select-form-field";
+import faker from "faker";
+import uuid from "uuid";
 
 describe("SelectFormField", () => {
-    test.skip("TODO - https://github.com/AndcultureCode/AndcultureCode.JavaScript.React.Components/issues/26", () => {});
+    test("when default props, renders SelectFormField with label", () => {
+        // Arrange
+        const label = faker.random.words();
+        const selectLabel = faker.random.word();
+        const selectValue = faker.random.word();
+
+        // Act
+        const { getByText } = render(
+            <SelectFormField
+                onChange={() => {}}
+                label={label}
+                id={uuid()}
+                values={[{ value: selectValue, label: selectLabel }]}
+            />
+        );
+
+        // Assert
+        expect(getByText(label)).not.toBeNil();
+    });
+
+    test("when has errorsMessages prop, renders select form field with error message", () => {
```

Comment:
@SaidShah as with the other PR, please expand the arrangement when passing arrays to atleast have multiple values. to ensure the system under test handles those values